helm/dagster/schema/schema_tests/test_dagit.py

```diff
@@ -324,6 +334,55 @@ def test_dagit_workspace_external_configmap(deployment_template: HelmTemplate):
     )
 
 
+def test_dagit_workspace_servers(workspace_configmap_template: HelmTemplate):
+    helm_values = DagsterHelmValues.construct(
+        dagit=Dagit.construct(
+            workspace=Workspace(
+                enabled=True,
+                servers=[
+                    Server(
+                        host="example.com",
+                        port=443,
+                        name="Example",
+                    )
+                ],
+                externalConfigmap=None,
+            )
+        ),
+    )
+
+    [dagit_workspace_configmap] = workspace_configmap_template.render(helm_values)
+    assert (
+        dagit_workspace_configmap.data["workspace.yaml"]
+        == """load_from:\n  - grpc_server:\n      host: example.com\n      port: 443\n      location_name: Example\n"""
+    )
```

Comment:
can we parse this yaml and assert it against a dict to make this more legible?

Reply:
Addressed.